setup.py

```diff
@@ -112,7 +112,8 @@ def find_rel_test():
     zip_safe=False,
     install_requires=[
         'setuptools >= 0.6',
-        "vsc-base >= 2.4.18",
+        "vsc-base >= 2.5.0",
+        'humanfriendly',  # determine whether terminal supports ANSI color
```

Comment:
this should be moved down into `extras_require`, only the packages that are strictly required to make `eb` work  should be listed in `install_requires`